docs/_summary.md

```diff
@@ -34,6 +34,7 @@
     * [Customizing Functionality](custom_quantum_functions.md)
     * [Driver Installation with Zadig](driver_installation_zadig.md)
     * [Keymap Overview](keymap.md)
+    * [Docker Guide](getting_started_docker.md)
```

Comment:
Do we want to group docker and vagrant under a new sub heading here?

Reply:
II think that may be a good idea, yeah.

Reply:
If we don't do that this should move up so everything stays in alphabetical order.